src/plants/PoisonBerry.js

```diff
@@ -0,0 +1,11 @@
+import { Plant } from '../organisms/Plant';
+
+export class PoisonBerry extends Plant {
+  constructor(positionY, positionX, age = 0) {
+    super(99, positionY, positionX, age); // High strength to ensure it kills the animal
```

Comment:
In theory, an organism could be stronger than 99 by eating multiple guaranas. You could implement the `onEaten` method like in the Guarana class.